components/gitpod-db/src/typeorm/workspace-db-impl.ts

```diff
@@ -710,16 +711,19 @@ export class TypeORMWorkspaceDBImpl extends TransactionalDBImpl<WorkspaceDB> imp
 
     // Find the (last triggered) prebuild for a given commit
     public async findPrebuiltWorkspaceByCommit(
-        cloneURL: string,
+        projectId: string,
```

Comment:
A `cloneURL` was used across the board to identify the project. Since we now can have multiple projects per cloneURL this is no longer the case, so we pass on the projected instead. This is the majority of this changeset.